plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAMonitor.java

```diff
@@ -32,17 +33,23 @@
 import java.util.concurrent.ConcurrentHashMap;
 
 public class KVMHAMonitor extends KVMHABase implements Runnable {
+
     private static final Logger s_logger = Logger.getLogger(KVMHAMonitor.class);
-    private final Map<String, NfsStoragePool> _storagePool = new ConcurrentHashMap<String, NfsStoragePool>();
+    private final Map<String, NfsStoragePool> _storagePool = new ConcurrentHashMap<>();
```

Comment:
And if we start to normalize these variable names in this file?
Removing these underscore at the start of any variable. at least where some changes were needed

Reply:
you mean
```suggestion
    private final Map<String, NfsStoragePool> storagePool = new ConcurrentHashMap<>();
```
@RodrigoDLopez ;)
(will need work in the rest of the file!!!)